js/startup/mouseForwardBack.js

```diff
@@ -0,0 +1,26 @@
+import { remote } from 'electron';
+
+// handles mouse back/forward buttons
+
+export default function () {
+
+  // app-command works on Windows only
+  $(window).on("app-command", (e, cmd) => {
```

Comment:
If 'app-command' only works on Windows, wouldn't it be better wrap the 'app-command' call in a check to make sure we're on windows, so the code is not unnecessarily executed (and then the linux "if" could be an "else-if")?

Where is OSX handled? Does OSX have this functionality by default?

Reply:
This seemed ok to me as it just reacts to the "app-command" event which will be fired only on windows. It just registers the handler on startup that is possibly not used later. Also if this is maybe fixed in electron in a way this will be called on other platforms too it can be used here (not sure about this, see the discussion in the electron issue linked above).
I will change the comment to be more accurate.
I haven't found how to do this on OSX.